requirements.txt

```diff
@@ -345,6 +342,8 @@ graphql-core==3.2.5
     #   graphql-relay
 graphql-relay==3.2.0
     # via graphene
+greenlet==3.1.1
+    # via sqlalchemy
```

Comment:
is this personal to you?
if i run pip compile this gets removed

Reply:
```suggestion

```

I guess it is, although i have no idea why, i may remove it.